pkgs/development/python-modules/sotabenchapi/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  click,
+  requests,
+  tabulate,
+}:
+
+let
+  version = "0.0.16";
+  pname = "sotabenchapi";
+in
+buildPythonPackage {
+  inherit pname version;
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-thbVH4aLmEgi8K17PkmbUg4nHqGj+dEiXPDILjvQMzk=";
+  };
+
+  postPatch = ''
+    touch requirements.txt
+  '';
```

Comment:
You removed my comments ?

Reply:
Oh no, this is another file. The `requirements.txt` is missing too ?

Reply:
Maybe add a comment too. The reason for this patch is not obvious when you look at it.

Reply:
Will do now.